### PR TITLE
Explore Blog Bug Fix

### LINKED ISSUE #92: Explore Blogs Link

### PR DESCRIPTION
**Pull request by khaldoon9kh**
Story post now loads in HomePage.
Added Link to each Story card.
Fixed styling.

Photo:
image

### CODE REVIEW

File: src/components/ExploreBlogs/index.jsx [hunk 1]
```diff
@@ -2,79 +2,68 @@ import React from "react";
...
+  React.useEffect(() => {
+    const fetchPosts = async () => {
+      const data = await fetch(_URL);
+      const fetchedPosts = await data.json();
+      setPosts(fetchedPosts.slice(0, 3));
+    };
+    fetchPosts();
+  }, []);
+
+  console.log(posts);
```
Comment (fadime-ozdemir) [inline src/components/ExploreBlogs/index.jsx]: you forgot to delete the console log

**Review by fadime-ozdemir — CHANGES_REQUESTED** [review]

File: src/components/ExploreBlogs/index.jsx (hunk 1 above)
Comment (khaldoon9kh) [reply src/components/ExploreBlogs/index.jsx]: My bad!

File: src/components/ExploreBlogs/index.css [hunk 2]
```diff
@@ -13,17 +13,28 @@
   padding: 4rem 4rem;
 }
 
+.exploreBlogsBlog {
+  margin: 0.2rem;
+}
```
Comment (halitbatur) [inline src/components/ExploreBlogs/index.css:19]: new line here

File: src/components/ExploreBlogs/index.css [hunk 3]
```diff
@@ -13,17 +13,28 @@
...
+.exploreBlogsHeadingParagraph {
+  text-align: center;
+}
```
Comment (halitbatur) [inline src/components/ExploreBlogs/index.css:23]: new line

**Review by halitbatur — APPROVED** [review]
small changes, testes are not passing please check why

File: src/components/ExploreBlogs/index.css (hunk 2 above)
Comment (khaldoon9kh) [reply src/components/ExploreBlogs/index.css:19]: revised.

File: src/components/ExploreBlogs/index.css (hunk 3 above)
Comment (khaldoon9kh) [reply src/components/ExploreBlogs/index.css:23]: revised

**Comment by khaldoon9kh** [comment]
PTAL

File: src/components/ExploreBlogs/index.jsx [hunk 4]
```diff
@@ -2,79 +2,67 @@ import React from "react";
...
 
+  const [posts, setPosts] = React.useState([]);
+  const _URL =
+    "https://public-api.wordpress.com/wp/v2/sites/nobarriers650349093.wordpress.com/posts";
+
+  React.useEffect(() => {
+    const fetchPosts = async () => {
+      const data = await fetch(_URL);
+      const fetchedPosts = await data.json();
+      setPosts(fetchedPosts.slice(0, 3));
```
Comment (fadime-ozdemir) [inline src/components/ExploreBlogs/index.jsx]: Can you change the magic numbers?

**Review by fadime-ozdemir — APPROVED** [review]
just one small change

File: src/components/ExploreBlogs/index.jsx (hunk 4 above)
Comment (khaldoon9kh) [reply src/components/ExploreBlogs/index.jsx]: why changing them tho?
I sliced them in order to render only 3 posts in the homePage.

Comment (halitbatur) [reply src/components/ExploreBlogs/index.jsx]: @khaldoon9kh yes change these into consts there shouldn't be magic numbers in the code

Comment (khaldoon9kh) [reply src/components/ExploreBlogs/index.jsx]: Resolved

**Comment by khaldoon9kh** [comment]
PTAL

**Pull request merged**